lib/tasks/migrate/20230725053637_add_unmatched_to_project_medias.rake

```diff
@@ -0,0 +1,34 @@
+namespace :check do
+  namespace :migrate do
+    task add_unmatched_to_project_media: :environment do
+      started = Time.now.to_i
+      last_team_id = Rails.cache.read('check:migrate:add_unmatched_to_project_media:team_id') || 0
+      Team.where('id > ?', last_team_id).find_each do |team|
+        Version.from_partition(team.id).where(event_type: 'destroy_relationship')
+        .where("object LIKE '%confirmed_sibling%confirmed_sibling%'")
+        .find_in_batches(:batch_size => 2500) do |versions|
+          pm_ids = versions.map(&:associated_id).uniq
+          target_ids = Relationship.where(target_id: pm_ids).where('relationship_type = ?', Relationship.confirmed_type.to_yaml).map(&:target_id)
```

Comment:
The condition should be `where('relationship_type IN (?)', [Relationship.confirmed_type.to_yaml, Relationship.suggested_type.to_yaml])`, or something like that.